### PR TITLE
docs(v2): add Sass/SCSS in docs under the styling section

### PR DESCRIPTION
**Pull request by rlamana**
## Motivation

For those looking for Sass/SCSS support and to show how modular v2 is ;), I added a small section to the styling section on how to use Sass with docusaurus v2 via the unofficial plugin I created.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### CODE REVIEW

**Comment by yangshun** [comment]
Master is broken because of me. I'll fix it tonight, sorry about that.

File: website/versioned_docs/version-2.0.0-alpha.48/styling-layout.md [hunk 1]
```diff
@@ -131,3 +131,75 @@ The class names which will be processed by webpack into a globally unique class
 _This section is a work in progress._ [Welcoming PRs](https://github.com/facebook/docusaurus/issues/1640).\_
 
 :::
+
+### Sass/SCSS
```
Comment (lex111) [inline website/versioned_docs/version-2.0.0-alpha.48/styling-layout.md]: Do not make new changes to any versioned docs. Please move this section to `website/docs/styling-layout.md`.

Comment (rlamana) [reply website/versioned_docs/version-2.0.0-alpha.48/styling-layout.md]: Makes sense. I just clicked the `Edit this page` buttons without thinking too much about it. 
I updated the PR.

**Comment by yangshun** [comment]
Would you also mind adding your plugin to https://v2.docusaurus.io/docs/next/resources?

**Comment by rlamana** [comment]
@yangshun sure! I just updated the PR.

**Comment by rlamana** [comment]
@yangshun I added the plugin here: #2446

**Pull request merged**